packages/schematics/angular/ng-new/index_spec.ts

```diff
@@ -56,4 +56,24 @@ describe('Ng New Schematic', () => {
     const moduleContent = tree.readContent('/foo/src/app/app.module.ts');
     expect(moduleContent).toMatch(/declarations:\s*\[\s*AppComponent\s*\]/m);
   });
+
+  describe('empty option', () => {
+    it('should create an empty workspace', () => {
+      const options = { ...defaultOptions, empty: true };
+
+      const tree = schematicRunner.runSchematic('ng-new', options);
+      const files = tree.files;
+      expect(files.indexOf('/bar/angular.json')).toBeGreaterThanOrEqual(0);
+    });
+
+    it('should not create a project', () => {
```

Comment:
nit: I think I'd put both assertions in the same test case

Reply:
Agreed, updating.